### DISCUSSION TITLE
can you change the background?

### DISCUSSION THREAD

**Discussion by Tangy-the-cat** [RedShyGuy/Vapecord-ACNL-Plugin#39] [Q&A] [answered]
I noticed the .bmp files and I was wondering if there was a way to make our own backgrounds for the plugin?

**Comment by SirEnder125** [comment, marked as answer]
Yes, it is actually very easy. Just open the .bmp files in an image editor/paint app and edit them to your liking.
Then the plugin automatically uses them when ran. :)
